packages/serverless-workflow-diagram-editor/package.json

```
@@ -22,7 +22,8 @@
     "build:dev:linux:darwin": "mvn clean install -DskipTests -DskipITs=$(build-env global.build.testIT --not) -Pno-showcase && pnpm dist",
     "build:dev:win32": "pnpm powershell \"mvn clean install `-DskipTests `-DskipITs=$(build-env global.build.testIT --not) `-Pno-showcase\" && pnpm dist",
     "build:prod": "run-script-os",
-    "build:prod:linux:darwin": "pnpm lint && mvn clean install -DskipTests=$(build-env global.build.test --not) -DskipITs=$(build-env global.build.testIT --not) -Pno-showcase && pnpm dist",
+    "build:prod:darwin": "pnpm lint && mvn clean install -DskipTests=$(build-env global.build.test --not) -DskipITs=true -Pno-showcase && pnpm dist",
+    "build:prod:linux": "pnpm lint && mvn clean install -DskipTests=$(build-env global.build.test --not) -DskipITs=$(build-env global.build.testIT --not) -Pno-showcase && pnpm dist",
```

Ok, found the issue. The name of the env property changed. Here's the correct changes:

```suggestion
    "build:prod:linux:darwin": "pnpm lint && mvn clean install -DskipTests=$(build-env tests.run --not) -Dmaven.test.failure.ignore=$(build-env tests.ignoreFailures) -DskipITs=$(build-env integrationTests.run --not) -Pno-showcase && pnpm dist",
```

---

That's a classic example of semantic conflicts. They happen and it's very hard to predict them. I'm glad we found it quickly :) Thanks a lot @Josephblt @romartin